src/db_rows.rs

```diff
@@ -91,6 +91,11 @@ pub struct NftMetadata {
     pub mb_internal_id: Option<BigDecimal>,
     pub price: Option<BigDecimal>,
     pub minters_allowlist: Option<Vec<String>>,
+    pub royalties: Option<serde_json::Value>,
+    pub royalty_percent: Option<BigDecimal>,
+    pub max_supply: Option<BigDecimal>,
+    pub last_possible_mint: Option<NaiveDateTime>,
+    pub is_locked: Option<bool>,
 }
 
 impl NftMetadata {
```

Comment:
Isn't this about the same as derive::Default?

In or you might be able to remove several lines of code here with `..default::Default()` (at least for all the Options).

Unless we are trying to be explicit.

Reply:
Right, gonna fix